assets/blocks/reader-registration/view.js

```diff
@@ -53,5 +65,45 @@ import './style.scss';
 				res.json().then( ( { message, data } ) => endLoginFlow( message, res.status, data ) );
 			} );
 		} );
+
+		const googleLogin = container.querySelector( '#newspack-google-login' );
```

Comment:
It's possible for multiple login forms to exist on the same page, e.g. in a prompt and within page content. We should change this to handle `container.querySelectorAll( '.newspack-google-login' );` (note it's a class name instead of an ID) and use `addEventListener` instead of `onclick` in case we need to add additional event handlers or have greater control over adding/removing this one.

Reply:
Done in 51ce912ecfec7ef3d2f38fbcf2ebc9570ece6dfe & 5030ffe46d38975f1d0a6bb5989e899e6b5b2842

Reply:
This will still only attach the event listener and flow for the first instance of the Registration block on the page. This should really be a `document.querySelectorAll`, and then a `for` loop to attach the event listener to each matching element.

Reply:
Given that the selector is queried from `container`, it's not the case. The `container` will always have only one `googleLogin`.

I would, though, avoid using an ID for this. If the page happens to render multiple blocks it'll create an ID conflict.

Reply:
> Given that the selector is queried from `container`, it's not the case. The `container` will always have only one `googleLogin`.

You're absolutely right! 51ce912 & 5030ffe did in fact change this ID to a class name, so I think we're good to go here.